extensions/amp-story/1.0/media-pool.js

```diff
@@ -516,6 +517,11 @@ export class MediaPool {
       new SwapIntoDomTask(placeholderEl)
     ).then(
       () => {
+        if (placeholderEl.isBitrateManaged) {
+          const bitrateManager = getBitrateManager(this.win_);
+          bitrateManager.unmanage(placeholderEl);
```

Comment:
I think in our current architecture, you'd want to create two new MediaTasks (manage and unmanage bitrate management), then enqueue those tasks here.  Having a single task queue for these helps us ensure that the media operations we care about all go through a centralized code path and happen in the order we want them to.